pandas_summary/__init__.py

```diff
@@ -252,7 +252,27 @@ def _get_column_summary(self, column):
             return self._get_date_summary(column)
         if column_type == self.TYPE_CONSTANT:
             return self._get_constant_summary(column)
-
+        
+    @property
+    def get_constants(self):
+        return self.df.columns[self.columns_stats.loc['types'] == 'constant']
+    @property
+    def get_categoricals(self):
+        return self.df.columns[self.columns_stats.loc['types'] == 'categorical']
+    @property
+    def get_numerics(self):
+        return self.df.columns[self.columns_stats.loc['types'] == 'numeric']
+    @property
+    def get_uniques(self):
+        return self.df.columns[self.columns_stats.loc['types'] == 'unique']
+    @property
+    def get_bools(self):
+        return self.df.columns[self.columns_stats.loc['types'] == 'bool']
+    
+    @property
+    def get_missing_frac(self):
+        return self.columns_stats.loc['missing'].apply(lambda x: float(x)/self.length)
+    
```

Comment:
I've added one more property which gives fraction of missing values. It was there in the form of a summary, but I felt this is a useful function for a lot of scenarios